ScreenControl_Unity/Assets/ScreenControl/Client/Scripts/Connection/ServiceConnection.cs

```diff
@@ -145,5 +145,39 @@ internal void SendMessage(string _message, string _requestID, Action<WebSocketRe
 
             webSocket.Send(_message);
         }
+
+        // Function: SendMessage
+        // Used internally to send or request information from the Service via the <webSocket>. To
+        // be given a pre-made _message and _requestID. Provides an asynchronous <ConfigStateResponse>
+        // via the _callback parameter.
+        internal void SendMessage(string _message, string _requestID, Action<ConfigStateResponse> _callback)
```

Comment:
Could we name this (and the original) differently to distinguish the messages being sent?